task2/Interfaces/IMovable.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace task2.Interfaces
+{
+    interface IMovable
+    {
+        void move();
```

Comment:
methods in C# are in [PascalCase](https://www.theserverside.com/definition/Pascal-case#:~:text=Pascal%20case%20%2D%2D%20or%20PascalCase,names%20to%20include%20blank%20spaces.)